cli/internal/runcache/runcache.go

```diff
@@ -225,8 +228,10 @@ func (tc TaskCache) OutputWriter() (io.WriteCloser, error) {
 	if err != nil {
 		return nil, err
 	}
+	colorPrefixer := tc.rc.colorCache.PrefixColor(tc.pt.PackageName)
```

Comment:
Probably not worth fixing right now, but if we end up revamping how we do coloring and logging, I think it would be good to have this prefix generation done once per task.

We also have a task somewhere to stop baking prefix (and color) into log files, but that's definitely beyond the scope here.